src/01-highlights-list/editor.scss

```diff
@@ -1,10 +1,9 @@
 @charset "UTF-8";
 /*********************************************************************************************
 
-Plugin Name: MOJ Blocks
-Block: Accordion (Gutenberg editor styles)
-Text Domain: mojblocks
+Name: Highlights List
+Author:  Ministry of Justice - Damien Wilson
 
```

Comment:
Yes, that accordion had to go. Wonder if we still should keep reference that it is a Gutenberg editor stylesheet? And btw, I don't mind if you want to put your name on individual blocks but I'm probably not going to be updating every block I work on with my name.